WrightTools/artists/_interact.py

```diff
@@ -153,16 +167,6 @@ def gen_ticklabels(points, signed=None):
     return ticklabels
 
 
-def norm(arr, signed, ignore_zero=True):
-    if signed:
-        norm = np.nanmax(np.abs(arr))
-    else:
-        norm = np.nanmax(arr)
-    if norm != 0 and ignore_zero:
-        arr /= norm
-    return arr
-
-
 def interact2D(
```

Comment:
where did norm go?

Reply:
this function was replaced by `Norm.__call__`